src/list.js

```diff
@@ -28,4 +24,36 @@ export default class ListOfTasks {
     this.tasksList[index].done = !this.tasksList[index].done;
     return this.tasksList;
   }
+
+  addToList(descr, index = this.tasksList.length, done = false) {
+    this.tasksList.push(new Task(descr, index, done));
+    Storage.saveAndUpdate(this.tasksList);
+    // Interact.populateUlTasksList(new ListOfTasks());
+  }
+
+  overrideIndexes(list) {
+    this.list = list;
+    this.list.forEach((task, i) => {
+      task.index = i;
+    });
+
+    return this.list;
+  }
+
+  removeAllDone() {
+    const filtered = this.tasksList.filter((task) => task.done === false);
+    Storage.saveAndUpdate(this.overrideIndexes(filtered));
+    // Interact.populateUlTasksList(new ListOfTasks());
+  }
+
+  removeByIndex(ind) {
+    const filtered = this.tasksList.filter((task) => task.index !== ind);
+    Storage.saveAndUpdate(this.overrideIndexes(filtered));
+    // Interact.populateUlTasksList(new ListOfTasks());
+  }
```

Comment:
- [ ] [OPTIONAL] There is an issue when removing a single element, whenever I have 2 tasks and I select the last one and then the first one and delete the first one, both tasks disappear. 🤨 
    - Here is a gif to show you exactly what happens. 🚀 
    ![issue](https://user-images.githubusercontent.com/60789317/134713407-ec306b01-dd3a-4d3d-a290-9043f2b6cfbb.gif)